data/topic-schema/src/main/resources/net/corda/schema/Services.yaml

```diff
@@ -24,8 +27,10 @@ topics:
     name: services.token.event.dlq
     consumers:
       - flow
+      - db
     producers:
       - flow
```

Comment:
flow won't be a producer for DLQ topic

Reply:
OK. I thought this list of consumers and producers should match with the one in the topic token.event.
What is the DLQ topic used for?